themes/react-data-grid-cell.css

```diff
@@ -27,6 +27,15 @@
   box-shadow: 2px 0 5px -2px rgba(136, 136, 136, .3) !important;
 }
 
+/* cell which have tooltips need to have a higher z-index on hover so that the tooltip appears above the other cells*/
+.react-grid-Cell.has-tooltip:hover {
+  z-index: 5;
```

Comment:
We need to change the values to match the new zIndex values
https://github.com/adazzle/react-data-grid/blob/master/packages/common/constants/zIndexes.js#L3

so 
```
z-index: 1;

and

z-index: 3
```